docs/source/configuration/external.mdx

```diff
@@ -327,12 +327,14 @@ For example, if the co-processor updated a received request so that the control
   "stage": "RouterRequest",
   "control": { "Break": 403 },
   "id": "1b19c05fdafc521016df33148ad63c1b",
-  "body": "You are not allowed to do this",
+  "body": {
+    "errors": [{ "message": "You are not allowed to do this" }]
+   },
   (etc...)
 }
 ```
 
-then the router would stop processing the request and return a response to the client with an HTTP status code of 403 and the error message "You are not allowed to do this" (read from the body).
+then the router would stop processing the request and return a response to the client with an HTTP status code of 403 and graphql response defined in the body.
```

Comment:
```suggestion
then the router would stop processing the request and return a response to the client with an HTTP status code of 403 and GraphQL response defined in the body.
```

Reply:
discarding in favor of the docs followup